gke/main.tf

```diff
@@ -53,7 +53,7 @@ resource "google_container_node_pool" "np" {
 
   management {
     auto_repair  = true
```

Comment:
Should change `auto_repair` to `false` as well as we don't want any disruptions in the middle of a run.

Reply:
done